plugins/woocommerce/README.md

```diff
@@ -12,22 +12,27 @@ If you are not a developer, please use the [WooCommerce plugin page](https://wor
 
 ## Getting Started
 
-Please make sure you follow the [repository's getting started guide](../../README.md#getting-started) first!
+### Prerequisites
+
+-   [Repository "Getting Started" Guide](../../README.md#getting-started)
+-   [Docker](https://docs.docker.com/get-docker/)
+
+Once you have completed the prerequisites, you can start the development environment:
 
 ```bash
-# Make sure that WooCommerce Core and all of its dependencies are built
-pnpm --filter=@woocommerce/plugin-woocommerce build 
-# Make sure you're in the WooCommerce Core directory
-cd plugins/woocommerce
+## Watch for changes in WooCommerce and all of its dependencies.
+pnpm --filter='@woocommerce/plugin-woocommerce' watch:build
 # Start the development environment
-pnpm -- wp-env start
+pnpm --filter='@woocommerce/plugin-woocommerce' env:dev
```

Comment:
I'm not sure this is the most appropriate section for this, but I wonder if it's worth noting that this step is optional and any environment can be used.  And if so, we might note the best process for using (symlinking) the project to a WP site.

Reply:
The placement looks good on my end, but it's a good point to expose the fact that multiple options do exists for running the development instance. Updated in 2dffdc6630b596fd4caca6b26014a2b521a547bf.